src/index.html

```diff
@@ -14,105 +14,151 @@
+            <li class="card__tag-list__item">Tag 4</li>
+          </ul>
+        </section>
 
-      <section class="card" data-js="card">
-        <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
-        <h2>What can you catch but never throw?</h2>
-        <details>
-          <summary class="card__show-answer">Show Answer</summary>
-          <p class="card__show-answer__answer text-center">A cold 🤧</p>
-        </details>
-        <ul class="card__tag-list flex-se">
-          <li class="card__tag-list__item">Tag 1</li>
-          <li class="card__tag-list__item">Tag 2</li>
-          <li class="card__tag-list__item">Tag 3</li>
-          <li class="card__tag-list__item">Tag 4</li>
-        </ul>
-      </section>
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>What belongs to you, but other people use it more than you?</h2>
+          <details>
+            <summary class="card__show-answer">Show Answer</summary>
+            <p class="card__show-answer__answer text-center">Your name</p>
+          </details>
+          <ul class="card__tag-list flex-se">
+            <li class="card__tag-list__item">Tag 1</li>
+            <li class="card__tag-list__item">Tag 2</li>
+            <li class="card__tag-list__item">Tag 3</li>
+            <li class="card__tag-list__item">Tag 4</li>
+          </ul>
+        </section>
 
-      <section class="card" data-js="card">
-        <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
-        <h2>What belongs to you, but other people use it more than you?</h2>
-        <details>
-          <summary class="card__show-answer">Show Answer</summary>
-          <p class="card__show-answer__answer text-center">Your name</p>
-        </details>
-        <ul class="card__tag-list flex-se">
-          <li class="card__tag-list__item">Tag 1</li>
-          <li class="card__tag-list__item">Tag 2</li>
-          <li class="card__tag-list__item">Tag 3</li>
-          <li class="card__tag-list__item">Tag 4</li>
-        </ul>
+        <section class="card" data-js="card">
+          <i class="card__bookmark fa fa-3x fa-bookmark" data-js="bookmark"></i>
+          <h2>What do you call two witches who live together?</h2>
+          <details>
+            <summary class="card__show-answer">Show Answer</summary>
+            <p class="card__show-answer__answer text-center">Broommates 🧹</p>
+          </details>
+          <ul class="card__tag-list flex-se">
+            <li class="card__tag-list__item">Tag 1</li>
+            <li class="card__tag-list__item">Tag 2</li>
+            <li class="card__tag-list__item">Tag 3</li>
+            <li class="card__tag-list__item">Tag 4</li>
+          </ul>
+        </section>
       </section>
+      <section class="page create-page hidden">
+        <form class="create-page__form">
+          <label class="create-page__form__label"
```

Comment:
There should not be nested elements in BEM. You are doing "BEEM" here (two Elements). 

Better start a new component "`form`" that has a "`form__label`". No reference for the page type needed.